client/me/purchases/downgrade/index.tsx

```diff
@@ -129,6 +130,27 @@ export const Downgrade: React.FC< DowngradeProps > = ( props ) => {
 	const isAtomicSiteDowngrade =
 		isAtomicSite && ! targetPlan?.getIncludedFeatures?.().includes( WPCOM_FEATURES_ATOMIC );
 
+	const refundAmount = useMemo( () => {
+		if ( ! purchase ) {
+			return '0';
+		}
+
+		const { refundOptions, currencyCode } = purchase;
+		const defaultFormatter = new Intl.NumberFormat( 'en-US', {
+			style: 'currency',
+			currency: currencyCode,
+		} );
```

Comment:
We should use `import { formatCurrency } from 'i18n-calypso';`

Reply:
This formatter is used to determine the `maximumFractionDigits`.
We do use this `formatCurrency` when printing the final amount value.